packages/rx_bloc_cli/mason_templates/bricks/feature_social_login_fb/__brick__/lib/lib_auth_facebook/models/facebook_auth_request_model.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'facebook_auth_request_model.g.dart';
+
+@JsonSerializable(includeIfNull: false)
+class FacebookAuthRequestModel {
+  final String? email;
+  final String? facebookToken;
+  final bool isAuthenticated;
+
```

Comment:
If we receive more user data ( sutch as name, email ...) from fb, please add it as well. We would send as mush user related data as possible to our server.